R/tm_g_distribution.R

```diff
@@ -595,7 +595,17 @@ srv_distribution <- function(id,
           }
 
         params_vals <- unname(params)
-        params_names <- names(params)
+        if (length(input$t_dist) != 0) {
+          map_distr_nams <- list(
+            normal = c("mean", "sd"),
+            lognormal = c("meanlog", "sdlog"),
+            gamma = c("shape", "rate"),
+            unif = c("min", "max")
+          )
+          params_names <- map_distr_nams[[input$t_dist]] %||% names(params)
+        } else {
+          params_names <- names(params)
+        }
```

Comment:
```suggestion
        map_distr_nams <- list(
          normal = c("mean", "sd"),
          lognormal = c("meanlog", "sdlog"),
          gamma = c("shape", "rate"),
          unif = c("min", "max")
        )
        params_names <- map_distr_nams[[input$t_dist]] %||% names(params)
```

Since you already using the `%||%` operator maybe we don't need the `if..else..` statement?
I tested this locally and seems to be working as expected.

Reply:
I wanted to prevent `input$t_dist` being NULL, but will double check how it looks without the `if`

```r
> x <- list(a = 1, b = 2)
> x[[NULL]]
Error in x[[NULL]] : attempt to select less than one element in get1index
```